momento-sdk/src/main/java/momento/sdk/SimpleCacheClient.java

```diff
@@ -168,10 +159,6 @@ public ListSigningKeysResponse listSigningKeys(Optional<String> nextToken) {
    * @param key The key to get
    * @return {@link CacheGetResponse} containing the status of the get operation and the associated
    *     value data.
-   * @throws momento.sdk.exceptions.PermissionDeniedException
-   * @throws ClientSdkException if key is null
-   * @throws NotFoundException
-   * @throws momento.sdk.exceptions.InternalServerException
    */
   public CacheGetResponse get(String cacheName, String key) {
     return scsDataClient.get(cacheName, key);
```

Comment:
Note: in the dotnet SDK when we were ramping up for 1.0 release, we decided to entirely remove the synchronous/blocking API.  I think we should consider this for Java too; I'm leaning towards that approach.  I think it would be helpful to have a few things in the examples dir that show how to block on the future from an async call to demonstrate that users can use it in a synchronous fashion without too much ceremony if they like, but beyond that I think we should probably just rip out all of the synchronous signatures.

If you agree with this, I would suggest floating that idea for feedback in #engineering.

Reply:
I wouldn't mind removing the synchronous methods.

Reply:
okay let's start a thread about this in #engineering today.  if we decide to do it, it can be a separate PR obvs.